src/react-apps/applications/service-development/src/features/deploy/deployReducer.ts

```diff
@@ -0,0 +1,55 @@
+import update from 'immutability-helper';
+import { Action, Reducer } from 'redux';
+import * as DeployActions from './deployActions';
+import * as DeployActionTypes from './deployActionTypes';
+
+export interface IDeployState {
+  deploymentList: any;
+  masterRepoStatus: any;
```

Comment:
No attribute for saving the error-message?

Reply:
I fix.

Reply:
Fixed.